tdrs-backend/tdpservice/settings/common.py

```diff
@@ -313,9 +313,9 @@ class Common(Configuration):
     CSP_CONNECT_SRC = ("'self'", "*.cloud.gov")
     CSP_MANIFEST_SRC = ("'self'")
     CSP_OBJECT_SRC = ("'none'")
-    CSP_OBJECT_SRC = ("'none'")
     CSP_FRAME_ANCESTORS = ("'none'")
-    CSP_FORM_ACTION = ("'none'")
+    CSP_FORM_ACTION = ("'self'")
+    CSP_STYLE_SRC = ("'self'", "'unsafe-inline'")
```

Comment:
are we planning to address the "new" [vulnerability](https://10008-281707402-gh.circle-artifacts.com/0/tdrs-backend/reports/owasp_report.html#10055)this introduces alongside the resolution to spike #1208 or do we need a separate follow-on? @abottoms-coder @jorgegonzalez